README.md

```diff
@@ -1,50 +1,774 @@
 # node-tar
 
-Tar for Node.js.
+[Fast](./benchmarks) and full-featured Tar for Node.js
 
-[![NPM](https://nodei.co/npm/tar.png)](https://nodei.co/npm/tar/)
+The API is designed to mimic the behavior of `tar(1)` on unix systems.
+If you are familiar with how tar works, most of this will hopefully be
+straightforward for you.  If not, then hopefully this module can teach
+you useful unix skills that may come in handy someday :)
 
-## API
+## Background
 
-See `examples/` for usage examples.
+A "tar file" or "tarball" is an archive of file system entries
+(directories, files, links, etc.)  The name comes from "tape archive".
+If you run `man tar` on almost any Unix command line, you'll learn
+quite a bit about what it can do, and its history.
 
-### var tar = require('tar')
+Tar has 5 main top-level commands:
 
-Returns an object with `.Pack`, `.Extract` and `.Parse` methods.
+* `c` Create an archive
+* `r` Replace entries within an archive
+* `u` Update entries within an archive (ie, replace if they're newer)
+* `t` List out the contents of an archive
+* `x` Extract an archive to disk
 
-### tar.Pack([properties])
+The other flags and options modify how this top level function works.
```

Comment:
Yeah, I focused on coverage and completeness rather than usability.  Right choice for a rough draft, imo, just to get all the material on the table, but I agree it could definitely use some reworking from someone not as in the weeds as I was when I wrote it.